### PR TITLE
workflow-execution: start RWE_ on Kubernetes

### PR DESCRIPTION
**Pull request by diegodelemos**
Connect #158.
Depends on https://github.com/reanahub/reana-commons/pull/75.

### CODE REVIEW

File: reana_workflow_controller/config.py [hunk 1]
```diff
@@ -50,3 +50,10 @@
 WORKFLOW_QUEUES = {'cwl': 'cwl-default-queue',
                    'yadage': 'yadage-default-queue',
                    'serial': 'serial-default-queue'}
+
+WORKFLOW_ENGINE_SHARED_VOLUME = 'vm'
```
Comment (tiborsimko) [inline reana_workflow_controller/config.py]: Hmm, `vm` may be perhaps confusion-prone, what about using `local`? (or perhaps `host`?)

Comment (diegodelemos) [reply reana_workflow_controller/config.py]: Thanks! I think host is the most representative, I was not sure what to use here 😕

File: reana_workflow_controller/config.py [hunk 2]
```diff
@@ -50,3 +53,23 @@
 WORKFLOW_QUEUES = {'cwl': 'cwl-default-queue',
                    'yadage': 'yadage-default-queue',
                    'serial': 'serial-default-queue'}
+
+REANA_STORAGE_BACKEND = os.getenv('REANA_STORAGE_BACKEND', 'local')
+"""Type of storage attached to the engines, one of ['local', 'ceph']."""
+
+MANILA_CEPHFS_PVC = 'manila-cephfs-pvc'
+"""If CEPH storage backned is used, this represents the name of the
```
Comment (okraskaj) [inline reana_workflow_controller/config.py]: typo in the docstring:
ba**ckn**ed where it should be ba**cke**nd

File: reana_workflow_controller/workflow_run_managers/kubernetes/__init__.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            "hostPath": {
+                "path": SHARED_FS_MAPPING['MOUNT_SOURCE_PATH'],
+            }
+        }
+    }
+    """Configuration to connect to the different storage backends."""
+
+    def start_batch_workflow_run(self):
+        """Start a batch workflow run."""
+        namespace = "default"
```
Comment (okraskaj) [inline reana_workflow_controller/workflow_run_managers/kubernetes/__init__.py]: I think here we should have single quotes

Comment (diegodelemos) [reply reana_workflow_controller/workflow_run_managers/kubernetes/__init__.py]: Regarding this, there is already a mix in the code base what do you think about using something like https://github.com/ambv/black? @reanahub/developers

Comment (tiborsimko) [reply reana_workflow_controller/workflow_run_managers/kubernetes/__init__.py]: "Black prefers double quotes over single quotes" :wink:

**Pull request merged**